build_tools/scripts/run_markdownlint.sh

```diff
@@ -20,10 +20,13 @@ declare -a included_files_patterns=(
 )
 
 declare -a excluded_files_patterns=(
-  "/third_party/"
+  "**/third_party/**"
   "**/node_modules/**"
 )
 
+# ${excluded_files_patterns} is expanded into
+# "--ignore pattern1 --ignore pattern2 ...", since markdownlint doesn't accept
+# "--ignore pattern1 pattern2 ...".
 markdownlint "${included_files_patterns[*]}" \
     --config ./docs/.markdownlint.yml \
-    --ignore "${excluded_files_patterns[*]}"
+    ${excluded_files_patterns[*]/#/--ignore }
```

Comment:
TIL: https://stackoverflow.com/questions/20366609/prefix-and-postfix-elements-of-a-bash-array

```console
$ ARRAY=( one two three )
$ echo ${ARRAY[*]/#/--test }
--test one --test two --test three
```

Neat.

Reply:
Bash is full of such things